ui/app/templates/required_signoff_modal.html

```diff
@@ -0,0 +1,80 @@
+<div class="modal-header">
+  <h3 class="modal-title" ng-bind="getTitle()">
+  </h3>
+</div>
+<div class="modal-body">
+  <form role="form">
+    <div class="form-group" ng-class="{'has-error': errors.product}" ng-show="!editing">
+      <label for="id_product">Product</label>
+      <input tabindex="1" type="text" class="form-control" id="id_product" ng-model="product" autocomplete="off"
+             typeahead="product for product in products | filter:$viewValue | limitTo:16">
+      <p class="help-block" ng-show="errors.product">{{ errors.product.join(", ") }}</p>
+    </div>
+    <div class="tabbable tabs-below" ng-show="!editing">
+      <ul class="nav nav-pills" role="tablist">
```

Comment:
I think I'm improved this slightly. I couldn't find something that looked awesome, but having some separator is better than nothing.